packages/office-addin-debugging/src/start.ts

```diff
@@ -203,11 +270,13 @@ export async function startDebugging(manifestPath: string, appType: AppType, app
         // live reload can only be enabled for the desktop app type
         // when using proxy debugging and the packager
         const canEnableLiveReload: boolean = isDesktopAppType && isProxyDebuggingMethod && !!packagerCommandLine;
+        // only enable live reload if it can be enabled
+        const liveReload = enableLiveReload && canEnableLiveReload;
```

Comment:
Is there a reason to rename this from `enableLiveReload` to just `liveReload`? I'd prefer to keep the original name unless some reason to rename it.

Reply:
Yes, the original code overwrote the input variable called `enableLiveReload` generally I try to avoid overwriting input variables because it can make it confusing to debug later.

Reply:
Thanks for clarifying that. I agree with your rationale.